log/log.go

```diff
@@ -123,6 +125,36 @@ func Error(ctx context.Context, fields map[string]interface{}, format string, ar
 			if err == nil {
 				entry = entry.WithField("identity_id", identityID)
 			}
+
+			if req := goa.ContextRequest(ctx); req != nil {
```

Comment:
We will add to the error message the necessary details to trace the issue. We don't need to add a request complete log message and with it these details. On the other hand, we will get this log not with any error but with those which has a goa context and contains the values of a http request. Generally, we should only log each error a single time.